docs/recipes/Adding-Styles.md

```diff
@@ -2,6 +2,52 @@
 
 # Adding Styles
 
-Recipe is forthcoming.
+## Adding a CSS File to Root Module
 
-[☝️ Return To Top](#adding-styles)
\ No newline at end of file
+In order to access styles within your application, you will need to add the stylesheet to the `<head>`. You can achieve this by using [react-helmet](https://github.com/nfl/react-helmet).
+
+```jsx
+import React, { Fragment } from 'react';
+import { Helmet } from 'react-helmet';
+
+const MyRoot = ({ children }) => (
+  <Fragment>
+    <Helmet>
+      <link rel="stylesheet" href="https://www.example.com/styles.min.css" />
```

Comment:
@JamesSingleton Maybe include a reminder that the domain has to be added to the `csp`? or is it implied?

Reply:
I can mention that.